.gitignore

```diff
@@ -1,2 +1,17 @@
+# Generic temp files
```

Comment:
This list is endless. Developers should use global gitignore config for generic stuff like that.
http://egorsmirnov.me/2015/05/04/global-gitignore-file.html